added akamai-case-study-Martin-Jenssen.md (#163)
NoroffFEU/development-platforms: src/content/docs/akamai/akamai-case-study-Martin-Jenssen.md

Reviewer: Add a reference to an AI tool if you used it.

More suggestions and comments inline below.

---
title: Akami Case Study
author: Martin Jenssen <Martinjens>
tags: Akami, Development Platforms, CDN, Server
---

## Introduction

Akamai serves as an intelligent traffic manager in the cloud, ensuring the seamless operation of your business and accommodating the expansion of your user base. Through effective load balancing, it adeptly handles the demands of website and mobile performance.

By adopting an Internet-centric approach to global load balancing, Akamai enhances the reliability of user access to your website or IP applications. Unlike conventional hardware-based solutions confined to data centers, the fault-tolerant GTM service makes judicious routing decisions. These decisions rely on real-time data center performance metrics and global Internet conditions, directing user requests to the most suitable data center based on the optimal Internet route for that specific user in that moment.

## Brief History

> You may use a bullet point list or subheadings to structure a timeline of your chosen topic's history:

Reviewer: Remove this.

- 1997: The start of Akamai.

> - 1997: Entered the MIT Entrepreneurship Competition where was one of six finalists.

Reviewer: ```suggestion
- 1997: Entered the MIT Entrepreneurship Competition where it was one of six finalists.
```

> - 1998: Akamai got obtained exclusive license to certain intellectual property from MIT, and development efforts began fall the same year.

Reviewer: ```suggestion
- 1998: Akamai obtained exclusive license to certain intellectual property from MIT, and development efforts began in fall of the same year.
```

- 1999: Going from students working, to getting more experienced internet business professionals joining the team. Notably, Paul Sagan(former President of Time inc.).
- 1999: Launching commercial service. The first live traffic was a pixel buried deep in the Disney site.
- 2019: Streaming media readers chioce award 2019, Akamai Intelligent Edge Platoform wins for Content Delivery Network.

## Main Features

- Protect, visualize, and segment your assets in the data center, cloud, and hybrid cloud infrastructure.
- Fend off a wide range of web, application, and API attacks.
- Gain full visibility into your entire API estate with continuous detection and monitoring.
- Take action on bot activity at the edge server, forwarding only clean traffic to the origin.
- Intelligently optimize the quality, format, and size of your web images and videos.
- High availability and performance, and resilience against DDoS attacks.
- SaaS solution that can provide the scalable infrastructure needed to convert, manage, and engage millions of customers.
- Quickly detect and disrupt brand impersonation attempts.

## Market Comparison

> Here are some advantages and disadvantages with Akamai

Reviewer: ```suggestion
Here are some advantages and disadvantages with Akamai.
```

## Akamai strengths 
- Strong customer base and steady revenues 
- Diversified product portfolio allows it to serve a variety of markets
- Competitive edge through robust network capabilities
- Strong R&D backed by robust financials
- Over 5000 employees in the organization
- Their customers include Facebook, Apple, Bing, Twitter

> - Many acquisitions have made it in a strong position

Reviewer: ```suggestion
- Many acquisitions have put it in a strong position
```
